### PR TITLE
Corrected tank's sprites in Fire Watch mission.

### LINKED ISSUE #869: Fire Watch mission is broken
- You cannot build any units, as you've got no money.
- Fire Fighter Tank's icon doesn't match the used sprites, it's missing the barrel.

### PR DESCRIPTION
**Pull request by Dzierzan**
- Also removed WithAttackAnimation trait.
- And adjusted facing tolerance when shooting.

Closes #869

### CODE REVIEW

File: mods/hv/maps/firewatch/sequences.yaml [hunk 1]
```diff
@@ -9,9 +9,6 @@ waterball:
 
 watertank:
 	idle:
-		Filename: bits/tank13.png
-		Facings: 8
-	shoot:
```
Comment (Mailaender) [inline mods/hv/maps/firewatch/sequences.yaml:14]: No, this is on purpose. It should show a hose when firing.

Comment (Dzierzan) [reply mods/hv/maps/firewatch/sequences.yaml:14]: A hose? I didn't see that. How the hose is meant to be shown if it uses idle sprites?

File: mods/hv/maps/firewatch/rules.yaml [hunk 2]
```diff
@@ -47,12 +47,11 @@ WATERTANK:
 	AttackFrontal:
 		Voice: Action
 		TargetLineColor: 0000FF
+		FacingTolerance: 30
 	AttackMove:
 		TargetLineColor: 0000FF
 	AutoTarget:
 		ScanRadius: 10
-	WithAttackAnimation:
-		Sequence: shoot
```
Comment (Mailaender) [inline mods/hv/maps/firewatch/rules.yaml:55]: Not sure why this is not played anymore.

**Comment by Dzierzan** [comment]
Superseded by 2d069fe9bbcef30be4ba9f39c9410f9ac73b7ada

**Pull request closed without merging**